### PR TITLE
Update to the new paper theme colors

### PR DESCRIPTION
**Pull request by abdonrd**

### CODE REVIEW

File: paper-dialog-common.css [hunk 1]
```diff
@@ -9,15 +9,15 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
...
+  @apply(--paper-font-body1);
+  @apply(--shadow-elevation-16dp);
 
+  -webkit-overflow-scrolling: touch;
   background: var(--paper-dialog-background-color, --primary-background-color);
   color: var(--paper-dialog-color, --primary-text-color);
+  display: block;
+  margin: 24px 40px;
 
-  @apply(--paper-font-body1);
```
Comment (notwaldorf) [inline paper-dialog-common.css]: why do these need to move?

Comment (abdonrd) [inline paper-dialog-common.css]: Like the other Polymer elements, if we can (because is cascading), set the external `apply()`s at the top. I turn back?

Comment (notwaldorf) [inline paper-dialog-common.css]: i'm just worried that they'll have side effects that we're not testing. Order of css statements matters, and we don't have a standard about where these apply should actually go (that they go at the top is just anecdotal; clearly here they don't). I can't 100% guarantee without looking at each of these applies that they don't set (for example) a `display: relative` that the other `display: block` is now overriding.

I know this sounds annoying but it's a case of "if it ain't broke don't fix it"... :(

Comment (abdonrd) [inline paper-dialog-common.css]: Received! Don't worry.
Thank you @notwaldorf!

I turn back now. :sparkles:

**Comment by abdonrd** [comment]
@notwaldorf updated! :sparkles:

**Comment by notwaldorf** [comment]
@abdonrd all the `@apply` still seem sorted? :(

**Comment by abdonrd** [comment]
@notwaldorf sorry, I selected wrong changes to push. Fixed!

**Comment by notwaldorf** [comment]
Thanks so much! 🎉

**Pull request merged**